docs/source-app/examples/model_server_app/model_server_app_content.rst

```diff
@@ -0,0 +1,84 @@
+
+*********
+Objective
+*********
+
+Create a simple application that trains and serves a `Sklearn <https://scikit-learn.org/stable/>`_ machine learning model with `MLServer from SeldonIO <https://github.com/SeldonIO/MLServer>`_
+
+----
+
+*****************
+Final Application
+*****************
+
+Here is a gif of the final application built in this example.
+
+.. figure::  https://pl-flash-data.s3.amazonaws.com/assets_lightning/ml_server_2.gif
+
+----
+
+*************
+System Design
+*************
+
+In order to create such application, we need to build several components:
+
+* A Model Train Component that trains a model and provide its trained weights
+
+* A Model Server Component that serves to an API endpoint the model generated by the **Model Train Component**.
+
+* A Load Testing Component that test the model server works as expected. This could be used to CI/CD the performance of newly generated models (left to the users).
```

Comment:
```suggestion
In order to create such application, we need to build several components:

* A Model Train Component that trains a model and provides its trained weights

* A Model Server Component that serves as an API endpoint for the model generated by the **Model Train Component**.

* A Load Testing Component that tests the model server works as expected. This could be used to CI/CD the performance of newly generated models (left to the users).
```